content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -138,7 +138,7 @@ Run prisma generate to generate Prisma Client.
 
 ## Accessing environment variables from the schema
 
-Environment variables can be accessed using the `env` function:
+Environment variables help you to **keep secrets out of the schema file** while **improving it's portability**. They can be accessed using the `env` function:
```

Comment:
"improving its portability" could use an explanation, how does it do that?
(you can use the same schema for different environments for example)